### PR TITLE
ISPN-8155

### PR DESCRIPTION
**Pull request by wburns**
SingleClusterExecutorTest.testExecutorTriConsumerTimeoutException random
failures

* Make sure timeout service doesn't capture state transfer as well

https://issues.jboss.org/browse/ISPN-8155

### CODE REVIEW

File: core/src/test/java/org/infinispan/manager/AllClusterExecutorTest.java [hunk 1]
```diff
@@ -393,15 +393,20 @@ public void call() throws InterruptedException, ExecutionException, TimeoutExcep
...
             }
 
+            // These are to make sure the call to the scheduled executor doesn't overlap with others, so these should
+            // be unique
+            long crazyNumber = 84129912895471L;
+            TimeUnit unit = TimeUnit.DAYS;
+
             CompletableFuture<Void> future =
-                  executor(cm1).timeout(10, TimeUnit.MILLISECONDS).submitConsumer(m -> {
+                  executor(cm1).timeout(crazyNumber, unit).submitConsumer(m -> {
```
Comment (karesti) [inline core/src/test/java/org/infinispan/manager/AllClusterExecutorTest.java]: If the scheduled executor is a mock at this point, why is this overlapping with other tests ?

**Comment by tristantarrant** [comment]
Merged. I have not seen any more failures locally (and it was failing very often)

File: core/src/test/java/org/infinispan/manager/AllClusterExecutorTest.java (hunk 1 above)
Comment (wburns) [reply core/src/test/java/org/infinispan/manager/AllClusterExecutorTest.java]: It isn't other tests. The problem is that we started adding other internal caches and those use state transfer and will call into the mock. That is why this wasn't failing when I fixed this before :)

Comment (karesti) [reply core/src/test/java/org/infinispan/manager/AllClusterExecutorTest.java]: oh ok ! aniway, tristan has already merged 👍

**Pull request merged**